packages/core/strapi/lib/Strapi.js

```diff
@@ -454,6 +460,8 @@ class Strapi {
 
   async load() {
     await this.register();
+    // Swap type customField for underlying data type
+    convertCustomFieldType(this);
```

Comment:
Can you move this in the register function right after the loads ?

Reply:
It doesn't look like I can do it right after the loads since even though the plugin has loaded the register lifecycle hasn't run so the custom field doesn't exist and it throws an error. I think the earliest I can do it is right after: `await this.runLifecyclesFunctions(LIFECYCLES.REGISTER);` in the register function.